kornia/core/module.py

```diff
@@ -249,7 +119,7 @@ def to_tensor(self, x: Any) -> Tensor:
             return from_numpy(np.array(x)).permute(2, 0, 1).float() / 255  # type: ignore
         raise TypeError("Input type not supported")
 
-    def to_numpy(self, x: Any) -> np.array:  # type: ignore
+    def to_numpy(self, x: Any) -> "np.array":  # type: ignore
```

Comment:
Numpy has now built in typing https://numpy.org/devdocs/reference/typing.html#numpy.typing.NDArray